src/views/README.md

```diff
@@ -0,0 +1,27 @@
+# Views
+
+Views are top-level entry points into the app:
+
+* Each view can be used standalone.
+* Each view has a URL (see `src/router` for registration).
+* Some views can also be used as "tabs" in the "Workspace" view.
+
+The Workspace view is a special composite view which provides the flexible tab
+layout. Other views (e.g. the Tree, Table and Graph views) can be opened as
+tabs within the Workspace view.
+
+
+## Toolbars
+
+Use the `ViewToolbar` component for view-specific toolbars.
+
+
+## Components
+
+You may want to take the generic part of a view and turn it into a reusable
+component.
+
+E.G. the Tree *view* displays Cylc cycles/tasks/jobs in a collapsible hierarchy.
+It uses the Tree *component* which provides the generic tree logic e.g.
+indentation, expand/collapse, etc. This Tree *component* is also used by
+the Workflows *view*.
```

Comment:
To avoid confusion with the Workspace view which is a bit niche anyway.
```suggestion
the GScan component.
```

Reply:
I don't agree with this one.

Reply:
It's technically more correct, but take it or leave it

https://github.com/cylc/cylc-ui/blob/f6b2b31fcd0f7546eccbd78e26281888bce9ee8e/src/views/Workflows.vue#L18-L20